physionet-django/templates/home.html

```diff
@@ -15,71 +15,128 @@
 
 {# avoid using flexbody style #}
 {% block body %}
-<body>
+<body class="modern-ui">
   {% include "navbar.html" %}
 
-  {% if front_page_banner %}
-  <div class="alert alert-primary front-page-banner">
-    <ul class="list-unstyled front-page-banner">
-    {% for news in front_page_banner %}
-    <li>
-      <a href="{% url 'news_by_slug' news.slug %}">{{ news.title }}</a>
-    </li>
-    {% endfor %}
-    </ul>
+  <div class="banner relative" style="height: 650px; background-image: linear-gradient(to right, white 40%, rgba(255,255,255,0) 80%), url({% static 'images/banner.png' %});">
+    <div class="container absolute my-0">
+      <div class="banner__column d-flex flex-column justify-content-center h-100">
+        <h1 style="font-weight: 600;">Advancing Medical Research<br>Through Open Data</h1>
```

Comment:
The banner text and slogan should be managed as environment variables. None of this should be hardcoded in the template.